e2e/ui/spec/keys/multi-variant.js

```diff
@@ -73,32 +53,22 @@ describe('MultiVariant value type', () => {
       },
     };
 
-    browser.setValue(selectors.KEY_VALUE_TYPE_INPUT, 'string');
-    browser.click(selectors.ADD_RULE_BUTTON);
-    KeysPage.removeRuleCondition(1, 1);
-    clickComp('convert-to-multi-variant-button');
+    Key.withValueType('string');
 
-    Object.keys(args).forEach((key, i) => {
-      if (i > 1) clickComp('add-variant-button');
-      browser.setValue(sliderComp('legend-value-input', i + 1), key);
-      browser.setValue(sliderComp('legend-percent-input', i + 1), args[key]);
-    });
-    browser.setValue(identitySelector, 'other');
+    Rule.add().removeCondition().multiVariant().setValues(args).withIdentity('other');
```

Comment:
should be setValues, and setIdentity.
or set({identity:'other', values:'args'})